src/sentry/testutils/cases.py

```diff
@@ -2329,6 +2329,17 @@ def mock_event_links(self, timestamp, project_id, level, replay_id, event_id):
         return transform_event_for_linking_payload(replay_id, event)
 
 
+@pytest.mark.snuba
+@requires_snuba
+@pytest.mark.usefixtures("reset_snuba")
+class UptimeCheckSnubaTestCase(TestCase):
```

Comment:
for some reason calling snuba's `drop` test endpoint doesn't work, but not a big deal for now